packages/nodes-base/nodes/Airtop/actions/common/fields.ts

```diff
@@ -34,30 +34,20 @@ export const profileNameField: INodeProperties = {
 	name: 'profileName',
 	type: 'string',
 	default: '',
-	description:
-		'The name of the <a href="https://docs.airtop.ai/guides/how-to/saving-a-profile" target="_blank">Profile</a> to load or create',
-	hint: 'Name of the <a href="https://docs.airtop.ai/guides/how-to/saving-a-profile" target="_blank">Profile</a> to load into the session. Must consist only of alphanumeric characters and hyphen "-"',
+	description: 'The name of the Airtop profile to load or create',
+	hint: '<a href="https://docs.airtop.ai/guides/how-to/saving-a-profile" target="_blank">Learn more</a> about Airtop profiles',
+	placeholder: 'e.g. my-x-profile',
 };
 
 export const urlField: INodeProperties = {
 	displayName: 'URL',
 	name: 'url',
 	type: 'string',
 	default: '',
-	placeholder: 'https://google.com',
+	placeholder: 'e.g. https://google.com',
 	description: 'URL to load in the window',
 };
 
-export const autoTerminateSessionHint: NodeHint = {
```

Comment:
Should we move this to the description at the end of this file?
`Whether to terminate the session after the operation is complete. When disabled, you must manually terminate the session. By default, idle sessions timeout after 10 minutes.`
Not sure if this is too long though for a description.